extensions/tw-colorsearch/src/index.tsx

```diff
@@ -328,10 +353,11 @@ export default function Command() {
           accessories={[{ icon: Icon.Text, text: item.hex.toUpperCase() }]}
           actions={
             <ActionPanel>
-              <Action.CopyToClipboard title="Copy HEX" content={item.hex.toUpperCase()} />
-              <Action.CopyToClipboard title="Copy RGB" content={item.rgb} />
+              <Action.CopyToClipboard title="Copy Hex" content={item.hex.toUpperCase()} />
+              <Action.CopyToClipboard title="Copy Rgb" content={item.rgb} />
```

Comment:
The `RGB` should be upper cased. See https://en.wikipedia.org/wiki/RGB_color_model.

```suggestion
              <Action.CopyToClipboard title="Copy RGB" content={item.rgb} />
```